### DISCUSSION TITLE
[Feature] Support for time based operations for Jobs.

### DISCUSSION THREAD

**Discussion by imohitkr** [immich-app/immich#11867] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hey, 
Thank you for the great app. One feature that I would like to schedule jobs to run for like 4 hr every night from 3 am. 

Something like this is already supported by Jellyfin for their jobs would be cool to have it here as well. 

Thank you!

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#5498

**Reply by imohitkr** [reply]
Yeah, but there is no option in the UI right ? 

My request is for something like this, which we can configure from the UI. 
<img width="1028" alt="Screenshot 2024-08-17 at 12 49 34" src="https://github.com/user-attachments/assets/042981f7-9c8f-4f51-aec2-2b306eb3c406">

It is not needed immediately, but it will be very helpful once it is integrated.
